core/bounds.gms

```diff
@@ -288,6 +288,9 @@ loop(te$(sameas(te,"ngcc") OR sameas(te,"ngt") OR sameas(te,"gaschp")),
   vm_cap.lo("2020",regi,te,"1")$pm_histCap("2020",regi,te) = 0.95 * pm_histCap("2020",regi,te);
 );
 
+*** fix capacities for BECCS technologies to  zero in 2020
+vm_cap.fx("2020",regi,te,rlf)$(sameas(te,"bioigccc") OR sameas(te,"bioftcrec") OR sameas(te,"bioh2c") OR sameas(te,"biogasc")) = 0;
```

Comment:
We could avoid listing all BECCS technologies individually by using the already existing sets teBio and teCCS. Then this bound will automatically be applied also to BECCS technologies that might be added in the future. This would read then:
`vm_cap.fx("2020",regi,te,rlf)$(teBio(te) AND teCCS(te)) = 0;`

Reply:
thanks for figuring this out!